### ISSUE TITLE
errors with floater_convert

### ISSUE THREAD

**Issue by rabernat** [rabernat/floater#31]
cc: @geosciz, @nathanieltarshish  

I just tried running floater_convert with the new options on yeti with my Pacific sector data.

```bash
$ floater_convert --output_format netcdf --ref_time 1993-01-01 --step_time 86400 float_trajectories
```

I got this error:
```
Traceback (most recent call last):
  File "/u/6/r/ra2697/.conda/envs/xmitgcm_env/bin/floater_convert", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/hmt/sirius1/prv0/u/6/r/ra2697/floater/scripts/floater_convert", line 76, in <module>
    output_prefix=args.output_prefix)
  File "/hmt/sirius1/prv0/u/6/r/ra2697/floater/floater/utils.py", line 264, in floats_to_netcdf
    step_num = int(dfcs.time.values[0])//step_time
TypeError: unsupported operand type(s) for //: 'int' and 'str'
```

What is going on?

**Comment by rabernat** [comment]
FYI, this run is located in `/vega/physo/users/ra2697/global_lagrangian/run_fix_floater`

**Comment by geosciz** [comment]
@rabernat It seems that `floater_convert` read the input `86400` as **str** rather than **int**.

**Comment by nathanieltarshish** [comment]
I think that `step_time` may be getting parsed wrong as string rather than int. As a temp fix, `86400` is the default value, so I think it would work if you drop `--step_time 86400`.

**Comment by nathanieltarshish** [comment]
Sorry, I did not see your comment @geosciz before I posted mine.

**Comment by rabernat** [comment]
This scenario should have been covered by the tests. So you should fix the error and add more tests that cover all the possible command line options. In a new PR.

**Comment by rabernat** [comment]
closed by #33
